logging/logrus/entry.go

```diff
@@ -150,10 +151,10 @@ func (entry *Entry) Printf(format string, args ...interface{}) {
 }
 
 // Warnf logs a message at level Warn on the standard logger.
+//
+// Deprecated: use Warningf
 func (entry *Entry) Warnf(format string, args ...interface{}) {
-	if entry.lgEntry.Logger.Level >= lg.WarnLevel {
-		entry.lgEntry.WithFields(entry.fields).Warnf(format, args...)
-	}
+	entry.Warningf(format, args...)
```

Comment:
Calling `Warningf` from `Warn` will include additional level in the stack trace and report wrong line for `loc` tag.